### PR TITLE
Feature/store frontend

### PR DESCRIPTION
**Pull request by pschuermann97**
## Reason for this PR:

- Store Frontend can display dummy baskets correctly but has a data format mismatch with the basket backend.
- Backend data format for the basket needs to be updated (@this-kramer)
- for this reason, the current state of the store frontend needs to be merged to develop (although store frontend is unfinished)

## Checklist:

Select whatever applies:

- [x] Code Properly Formatted
- [x] Documented

### CODE REVIEW

File: services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java [hunk 1]
```diff
@@ -15,23 +15,46 @@
...
+    * This allows for the user to safely retry redeeming in case of connectivity problems
+    * without allowing for easy double-spending attacks.
+    */
     @ApiModelProperty(value = "${basketModel.redeemRequest}")
     private String redeemRequest;
-    // value must be set manually for serialization
+
     @ApiModelProperty(value = "${basketModel.value}")
-    private long value;
+    private long value; // value must be set manually for serialization
```
Comment (JanBobolz) [inline services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java:52]: I don't understand that comment.

**Review by JanBobolz — APPROVED** [review]

File: services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java (hunk 1 above)
Comment (this-kramer) [reply services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java:52]: Is outdated anyway :)

Comment (this-kramer) [reply services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java:52]: *will be  outdated with #178

**Review by this-kramer — APPROVED** [review]
Nice. 

Maybe add some newlines at the end of the files, but not that important

File: services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java (hunk 1 above)
Comment (pschuermann97) [reply services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java:52]: Me neither actually but I feel like it was not by me originally

**Pull request merged**